snippets/language-javascript.cson

```diff
@@ -37,16 +37,7 @@
     'body': 'function* ($1) {\n\t$2\n}'
   'getElementById':
     'prefix': 'get'
-    'body': 'getElementById(${1:\'${2:id}\'})'
-  'getElementsByClassName':
-    'prefix': 'get'
-    'body': 'getElementsByClassName(${1:\'${2:className}\'})'
-  'getElementsByName':
-    'prefix': 'get'
-    'body': 'getElementsByName(${1:\'${2:name}\'})'
-  'getElementsByTagName':
-    'prefix': 'get'
-    'body': 'getElementsByTagName(${1:\'${2:tagName}\'})'
+    'body': 'getElementById(${1:\'${2:id}\'})$3'
```

Comment:
What do you think about keeping all of these and just adding new prefixes instead?

* `geti` -> `getElementById`
* `getc` -> `getElementByClassName`
* `getn` -> `getElementsByName`
* `gett` ->  `getElementsByTagName`